- set version explicitly
- remove reference to brr.dev (managed by the port)

Index: slipshow.opam
--- slipshow.opam.orig
+++ slipshow.opam
@@ -1,6 +1,7 @@
 # This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
 synopsis: "A compiler from markdown to slipshow"
+version: 0.1.0
 description:
   "Slipshow is an engine to write slips, a concept evolved from slides."
 maintainer: ["Paul-Elliot"]
@@ -46,6 +47,3 @@ build: [
   ]
 ]
 dev-repo: "git+https://github.com/panglesd/slipshow.git"
-pin-depends: [
-  "brr.dev" "git+https://github.com/panglesd/brr.git#bindings-window-inner-size"
-]
